packages/react-components/react-dialog/src/components/Dialog/useDialog.ts

```diff
@@ -54,9 +58,20 @@ export const useDialog_unstable = (props: DialogProps): DialogState => {
     requestOpenChange,
   });
 
-  const handlebackdropClick = useEventCallback((event: React.MouseEvent<HTMLDivElement>) => {
+  const { targetDocument } = useFluent_unstable();
```

Comment:
if this is the only reason to use `useFluent` in the Dialog can't it be invoked directly in the `useDisableScroll` hook ? then you could just return a noop when document doesn't exist

Reply:
Yeah, I would also prefer that option.

Reply:
Would that bring any benefit? I mean, that would require me to change again the hook to be `useDisableBodyScroll` and I kind of feel like this way is a little bit more generic, but if there're any benefits then I'm all up for it....

Reply:
Sure, no problem. they're equivalent IMO, I'll see to that on a follow up PR together with avoiding jumps!